examples/file-uploader/js/list.js

```diff
@@ -0,0 +1,87 @@
+const Type = require('union-type');
+const T = require('ramda/src/T')
+    , adjust = require('ramda/src/adjust')
+    , append = require('ramda/src/append')
+    , curry  = require('ramda/src/curry')
+;
+const h = require('snabbdom/h');
+
+const upload = require('./upload');
+const uploader = require('./uploader');
+
+const sync = (s) => [s, []];
+
+// note: prefer to check if iterable, 
+// but FileList.prototype doesn't seem to have Symbol.iterator cross-browser?
+const isFileList = (x) => !(undefined === x.length) 
```

Comment:
Old yoda habit ;) . BTW I don't know if this comment about Symbol.iterator is accurate or not, it may be because I didn't include the babel polyfill for iterator functionality or something, didn't have time to chase it down.